source/framework/sensitivity/src/TRestComponent.cxx

```diff
@@ -292,23 +292,20 @@ ROOT::RDF::RNode TRestComponent::GetMonteCarloDataFrame(Int_t N) {
     ROOT::RDF::RNode df = ROOT::RDataFrame(N);
 
     // Function to fill the RDataFrame using GetRandom method
-    auto fillDataFrame = [this]() {
+   auto fillRndm = [this]() {
         auto randomValues = GetRandom();
+        return randomValues;
+   }
+   df.Define("Rndm", fillRndm)
```

Comment:
It needs a semi-colon at the end? 

I am a bit confused with lambda functions, so I am not sure if it is some new construction I am not aware of.